docs/docusaurus/docs/conceptual_guides/gx_overview.md

```diff
@@ -0,0 +1,118 @@
+- The Datasource and Data Asset configurations that tell GX how to connect to your Source Data System.
+- The Expectations you have specified about your Source Data.
+- The Metrics that are recorded when GX validates certain Expectations against data.
+- The Checkpoints you have configured for validating your Expectations.
+- The Validation Results that GX returns when you run a Checkpoint.
+
+A GX Data Context comes with default configurations in place for your Stores.  By default, these exist in the same folder hierarchy as your Data Context (or in memory, if you are using an Ephemeral Data Context).  However, you can update these if you want to change where your Store configurations exist, such as hosting them in a shared cloud environment like AWS.
+
+<!-- For more information on configuring Stores, see [our guides on metadata Stores](landing page: setting up a GX environment->Metadata Stores). -->
+
+**Data Docs** are human-readable documentation about your Expectations and Validation Results that GX generates for you.  They exist as webpages, and like Stores your Data Context will include a default configuration for building them locally for Filesystem Data Contexts, in memory for Ephemeral Data Contexts, and online for Cloud Data Contexts.  As with Stores, you can configure where your Data Docs are hosted.  You can also add configurations for multiple Data Docs sites, and specify what information each Data Docs site will present to its viewers.
+
+<!-- For more information on configuring Data Docs, see our [Data Docs guides](landing page:Setting up a GX environment->Data Docs). -->
+
+
+### Connect GX to your Source Data System
+
+**Datasources** provide the interface between GX and your Source Data System.  Regardless of the original format of your Source Data, a Datasource will provide you a clear and consistent API for accessing that data in GX.
+
+**Data Assets** are collections of records within a Datasource.  These collections can be further partitioned to a desired specification.  We call these subsets of data **Batches**.  For instance, you could define a Data Asset for a SQL Datasource as the selection of all records from last year in a given table.  Then you could further partition that Data Asset into Batches of data corresponding to the records for individual months of that year.
+
+How you organize your Datasources into Data Assets and your Data Assets into Batches will impact how you can analyze them with GX.  Defining multiple Batches in a Data Asset will let GX automate the process of running statistical analyzes for each Batch.  It will also let you specify an individual Batch to work with.  In the example of a Data Asset that has been split into months, this would let you build a statistical model off of each month of your existing data.  Then, after an update, you could specify that you only wanted to run your analysis on the most recent month's data.
```

Comment:
```suggestion
How you organize your Datasources into Data Assets and your Data Assets into Batches determines how you can analyze them with GX.  Defining multiple Batches in a Data Asset lets GX automate the process of running statistical analyzes for each Batch.  It also lets you specify an individual Batch to work with.  In the example of a Data Asset that has been split into months, this would let you build a statistical model for each month of your existing data.  Then, after an update, you could specify that you only wanted to run your analysis on the most recent month's data.
```

This is heavy going. Is there any way we can clarify and lighten it?